task/createPullRequest.ps1

```diff
@@ -741,64 +741,29 @@ function GetReviewerId() {
     
     # If it's Azure DevOps
     else {
-        $url = "$($env:System_TeamFoundationCollectionUri)_apis/userentitlements?top=10000&api-version=4.1-preview.1"
-        # Check if it's the old url or the new url, reltaed to issue #21
-        # And add "vsaex" to the rest api url 
-        if ($url -match "visualstudio.com") {
-            $url = $url.Replace(".visualstudio", ".vsaex.visualstudio")
-        }
-        else {
-            $url = $url.Replace("//dev", "//vsaex.dev")
-        }
-        $users = Invoke-RestMethod -Uri $url -Method Get -ContentType application/json -Headers $head
-        $teamsUrl = "$($env:System_TeamFoundationCollectionUri)_apis/projects/$($env:System_TeamProject)/teams?api-version=4.0-preview.1"
-        $teams = Invoke-RestMethod -Uri $teamsUrl -Method Get -ContentType application/json -Headers $head
+        $url = $($env:System_TeamFoundationCollectionUri).Replace("visualstudio.com", "dev.azure.com").Replace("dev.azure.com", "vssps.dev.azure.com") + "_apis/identities?api-version=7.0"
+        # API reference: https://learn.microsoft.com/en-us/rest/api/azure/devops/ims/identities/read-identities?view=azure-devops-rest-7.0&tabs=HTTP
         Write-Debug $reviewers
-        $split = $reviewers.Split(';').Trim()
         $reviewersId = @()
-        ForEach ($reviewer in $split) {
-            $isRequired = "false"
-            if ($reviewer -match "req:") {
-                $reviewer = $reviewer.Replace("req:","")
-                $isRequired = "true"
+        foreach ($reviewer in $reviewers.Split(';').Trim()) {
+            $isRequired = $reviewer -imatch "^req:"
```

Comment:
Hi, I tested it and you need to "reset" the $url here in the foreach, without it it will append all the reviewers to the url and return only 1 (I checked it with one user and one team).

Reply:
It was a mistake, thanks for catching!